src/tests/work-item/work-item-list/quickadd-workitem.spec.js

```diff
@@ -39,18 +38,17 @@ describe('Work item list', function () {
   });
 
   it('Creating a new quick add work item and Cancel delete - phone.', function () {
-    setBrowserMode('phone');
```

Comment:
My idea was to have a default screen size for all tests - and only change the size as needed for specific tests.